pandas/_libs/tslibs/period.pyx

```diff
@@ -1275,6 +1275,25 @@ cdef class _Period(object):
 
     @property
     def daysinmonth(self):
+        """
+        Get total days of month that a Period falls on.
```

Comment:
I would maybe rephrase this as "Get the total number of days of the month that the Period falls in"

Reply:
Sure will update now . As I'm not good in making sentences thanks for guiding :) :+1: